eis_toolkit/transformations/coda/alr.py

```diff
@@ -18,68 +20,68 @@ def _alr_transform(df: pd.DataFrame, columns: Sequence[str], denominator_column:
 
 
 @beartype
-@check_compositional
-def alr_transform(df: pd.DataFrame, idx: int = -1, keep_redundant_column: bool = False) -> pd.DataFrame:
+def alr_transform(df: pd.DataFrame, column: int = -1, keep_denominator_column: bool = False) -> pd.DataFrame:
     """
     Perform an additive logratio transformation on the data.
 
     Args:
         df: A dataframe of compositional data.
-        idx: The integer position based index of the column of the dataframe to be used as denominator.
+        column: The integer position based index of the column of the dataframe to be used as denominator.
             If not provided, the last column will be used.
```

Comment:
Did you leave the type of this parameter `int` on purpose? What I suggested was to let the user give the desired column name (str) instead of the index. This solution isn't bad, but can be a bit tricky if there are tens of columns in a DF :)

Reply:
I must have missed it or forgotten halfway through going through the change suggestions. 😄 It's now fixed!